packages/lab/src/multiline-input/MultilineInput.css

```diff
@@ -0,0 +1,245 @@
+  --multilineInput-background-disabled: var(--salt-editable-secondary-background-disabled);
+  --multilineInput-background-readonly: var(--salt-editable-secondary-background-readonly);
+}
+
+/* Style applied to input if `validationState="error"` */
+.saltMultilineInput-error,
+.saltMultilineInput-error:hover {
+  --multilineInput-background: var(--salt-status-error-background);
+  --multilineInput-background-active: var(--salt-status-error-background);
+  --multilineInput-background-hover: var(--salt-status-error-background);
+  --multilineInput-borderColor: var(--salt-status-error-borderColor);
+  --multilineInput-outlineColor: var(--salt-status-error-borderColor);
+}
+
+/* Style applied to input if `validationState="warning"` */
+.saltMultilineInput-warning,
+.saltMultilineInput-warning:hover {
+  --multilineInput-background: var(--salt-status-warning-background);
+  --multilineInput-background-active: var(--salt-status-warning-background);
+  --multilineInput-background-hover: var(--salt-status-warning-background);
+  --multilineInput-borderColor: var(--salt-status-warning-borderColor);
+  --multilineInput-outlineColor: var(--salt-status-warning-borderColor);
+}
+
+/* Style applied to input if `validationState="success"` */
+.saltMultilineInput-success,
+.saltMultilineInput-success:hover {
+  --multilineInput-background: var(--salt-status-success-background);
+  --multilineInput-background-active: var(--salt-status-success-background);
+  --multilineInput-background-hover: var(--salt-status-success-background);
+  --multilineInput-borderColor: var(--salt-status-success-borderColor);
+  --multilineInput-outlineColor: var(--salt-status-success-borderColor);
+}
+
+/* Style applied to inner textarea element */
+.saltMultilineInput-textarea {
+  background: none;
+  border: none;
+  box-sizing: content-box;
+  color: inherit;
+  cursor: inherit;
+  display: block;
+  flex: 1;
+  font: inherit;
+  letter-spacing: 0;
+  margin: 0;
+  min-width: 0;
+  overflow: hidden;
+  resize: none;
+  padding: var(--salt-spacing-100) 0;
+  text-align: var(--multilineInput-textAlign);
+  width: 100%;
+}
+
+/* Reset in the  class */
+.saltMultilineInput-textarea:focus {
+  outline: none;
+}
+
+/* Style applied to selected input */
+.saltMultilineInput-textarea::selection {
+  background: var(--salt-text-background-selected);
+}
+
+/* Styling when focused */
+.saltMultilineInput-focused {
+  --multilineInput-borderColor: var(--multilineInput-outlineColor);
+  --multilineInput-activationIndicator-borderWidth: var(--salt-editable-borderWidth-active);
+
+  outline: var(--salt-focused-outlineWidth) var(--salt-focused-outlineStyle) var(--multilineInput-outlineColor);
+}
+
+/* Styling when focused if `disabled={true}` or `readOnly={true}` */
+.saltMultilineInput-focused.saltMultilineInput-readOnly,
+.saltMultilineInput-focused.saltMultilineInput-disabled {
+  --multilineInput-activationIndicator-borderWidth: var(--salt-size-border);
+
+  outline: none;
+}
```

Comment:
this is applicable only when readOnly (no need for disabled). But should the readOnly be focusable (have focused class)?